demo/src/demo-vc.ts

```diff
@@ -1,5 +1,6 @@
 import * as Cord from '@cord.network/api'
 import { UUID } from '@cord.network/utils'
+import type { VerifiableCredential } from 'vc-export/src/types'
```

Comment:
Could be import from '@cord.network/vc-export'.

Not a blocker, as while taking this from here to cord-demo-scripts repo, it can be fixed.